backend/api/migrations/0043_ohshownevent_sight_see_date_time.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.13 on 2022-04-24 08:33
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0042_load_initial_data'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='ohshownevent',
+            name='sight_see_date_time',
+            field=models.DateTimeField(auto_now_add=True, default=django.utils.timezone.now),
```

Comment:
Ah, Looks like it's an issue if the `make migration` command will put this `auto_now_add=True` in the schema automatically.
@tai271828 @yanghaochang104

Reply:
This line differs from https://github.com/OH-SHOWN/ohshown-backend/pull/59#discussion_r864888617 since this line is used for generating fake testing data. `auto_now_add=True` is on purpose here.